### PR TITLE
Added link to Version Skew Policy in Upgrading Kubeadm Cluster

### PR DESCRIPTION
**Pull request by edoardo-c**
Hi,
This is nothing more than a link to another document that complements the "Upgrading Kubeadm Cluster". It should allow us to close [#33259](https://github.com/kubernetes/website/issues/33259).

### CODE REVIEW

**Comment by sftim** [comment]
https://github.com/kubernetes/website/pull/33293#issuecomment-1112562100 doesn't explain it very well but you need to make sure that the email address you used in the commit (`edoardo.costa@ssobuddy.be`) is one that GitHub recognizes as you (@edoardo-c). Once you've done that, and have signed the CLA, it should work.

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ weight: 20
 This page explains how to upgrade a Kubernetes cluster created with kubeadm from version
 {{< skew currentVersionAddMinor -1 >}}.x to version {{< skew currentVersion >}}.x, and from version
 {{< skew currentVersion >}}.x to {{< skew currentVersion >}}.y (where `y > x`). Skipping MINOR versions
-when upgrading is unsupported.
+when upgrading is unsupported. For more details, please visit [Version Skew Policy].(https://kubernetes.io/releases/version-skew-policy/).
```
Comment (neolit123) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:14]:
```suggestion
when upgrading is unsupported. For more details, please visit [Version Skew Policy](https://kubernetes.io/releases/version-skew-policy/).
```

has extra dot in the link.

Comment (edoardo-c) [reply content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:14]: Well spotted, thanks.

**Comment by edoardo-c** [comment]
@sftim , using the ssobuddy email address was a mistake of mine. I should not have used that one. I'm not sure how that happened. What would be best? Close this pull request and do it all over properly?

**Comment by divya-mohan0209** [comment]
@edoardo-c : You can do the forking, cloning etc all over again OR you can follow the steps below:

1. Set your local user.email & username by following the instructions [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-user-account/managing-email-preferences/setting-your-commit-email-address).
2. Amend the author of your last commit by using the command : git commit --amend --author="Author Name <email@address.com>" 
3. Perform a push with the force flag.

**Review by divya-mohan0209 — APPROVED** [review]
Approving from SIG Docs side. @neolit123 - PTAL & kindly lgtm it if this looks fine to you.

**Review by neolit123 — COMMENTED** [review]
Thanks
/label triage/merge-method-squash
/lgtm

**Pull request merged**